pom.xml

```diff
@@ -1678,6 +1681,44 @@
         </exclusions>
       </dependency>
       <dependency>
+        <groupId>org.apache.orc</groupId>
+        <artifactId>orc-core</artifactId>
+        <version>${orc.version}</version>
+        <classifier>${orc.classifier}</classifier>
+        <scope>${orc.deps.scope}</scope>
+        <exclusions>
+          <exclusion>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-storage-api</artifactId>
```

Comment:
Thank you so much for review, @cloud-fan .

- The original `orc-core-1.4.0.jar` has `hive-storage-api` dependency. ([Maven Repo](https://mvnrepository.com/artifact/org.apache.orc/orc-core/1.4.))
- `orc-core-1.4.0-nohive.jar` is a shaded jar file including `hive-storage-api` under `org.apache.orc` namespace.

`orc-core-1.4.0-nohive.jar` is designed for users and apps who don't want to depend on (or consider) `hive`. `nohive` is a classifier for this purpose.

This PR uses `orc-core-1.4.0-nohive` only. To avoid Maven confusion, this exclution makes it sure by removing the `hive-storage-api` dependency explicitly from `orc-core` artifact.